.travis.sh

```diff
@@ -187,6 +187,17 @@ then
          tests_sequence_unstable_target ${WATCHOS_SIM_ID} auto-watchos-x86_64-sim
      fi
      xcrun simctl delete $WATCHOS_SIM_ID
+
+     if [ "$(uname -m)" = "arm64" ]; then
+         title "••••• Darwin: visionOS simulator tests •••••"
+         title "boot a simulator"
+         VISIONOS_RUNTIME_ID=$(xcrun simctl list runtimes | grep visionOS | cut -d ' ' -f 7 | tail -1)
+         export VISIONOS_SIM_ID=$(xcrun simctl create My-apple-vision-pro com.apple.CoreSimulator.SimDeviceType.Apple-Vision-Pro $VISIONOS_RUNTIME_ID)
+
+         xcrun simctl boot $VISIONOS_SIM_ID
+         tests_sequence_unstable_target ${VISIONOS_SIM_ID} auto-watchos-aarch64-sim
+         xcrun simctl delete $VISIONOS_SIM_ID
+     fi
```

Comment:
There are now 4 simulator tests ran in a sequence. What do we think about putting them in a [github action matrix like this](https://github.com/liveview-native/liveview-native-core/blob/205796ec0464e7af11e221e765657b1698bccad1/.github/workflows/apple-devices.yml#L14-L35)?

Reply:
Probably a good idea as the macos steps are already what take the most time on the ci

Reply:
I did this in #228.